hack/development-environment/kind-local-up.sh

```diff
@@ -51,27 +75,38 @@ function install_k8s() {
 
 function install_calico() {
     kubectl get pods
-    pushd ${ROOT_CALICO_REPOS_DIR}/calico/_output/dev-manifests
-    	kubectl apply -f ./calico.yaml 
-    	kubectl get pods -n kube-system
-    popd
-    sleep 5 ; kubectl -n kube-system set env daemonset/calico-node FELIX_IGNORELOOSERPF=true
-    sleep 5 ; kubectl -n kube-system get pods | grep calico-node
-    echo "will wait for calico to start running now... "
-    while true ; do
-        kubectl -n kube-system get pods
-        sleep 3
-    done
+
+        pushd $ROOT_CALICO_REPOS_DIR/calico/_output/dev-manifests
+            kubectl apply -f ./calico.yaml 
+            kubectl get pods -n kube-system
+        popd
+        sleep 5 ; kubectl -n kube-system set env daemonset/calico-node FELIX_IGNORELOOSERPF=true
+        sleep 5 ; kubectl -n kube-system get pods | grep calico-node
+        echo "will wait for calico to start running now... "
+        while true ; do
+            kubectl -n kube-system get pods
+            sleep 3
+        done
```

Comment:
Here's what I use, in case you want to steal it:
```
    # Wait for installation to succeed and everything to be ready.
    for k8sapp in calico-node kube-dns calico-kube-controllers; do
	while ! time ${kubectl} wait pod --for=condition=Ready -l k8s-app=${k8sapp} -n kube-system --timeout=300s; do
	    # This happens when no matching resources exist yet,
	    # i.e. immediately after application of the Calico YAML.
	    sleep 5
	done
    done
```

Reply:
agree w/ you this is a little better do do the explicit wait pod ...  to get this to merge so we have a baseline that works though ill keep as is , and we can improve this logic significantly later ?

Reply:
ok